src/transformers/models/bloom/configuration_bloom.py

```diff
@@ -117,9 +114,7 @@ class BloomConfig(PretrainedConfig):
     keys_to_ignore_at_inference = ["past_key_values"]
     attribute_map = {
         "num_hidden_layers": "n_layer",
-        "n_head": "num_attention_heads",
-        "hidden_size": "n_embed",
```

Comment:
Those should not be removed.

Reply:
Why do we need to map `n_embed` to `hidden_size` if the kwarg in `__init___` is called `hidden_size`?
If I'm not mistaken the `"n_head": "num_attention_heads"` attribute map is the wrong way around, so I added it as `"num_attention_heads": "n_head"`